javascript-practice/src/js/views/loginView.js

```diff
@@ -0,0 +1,91 @@
+import CommonLoginRegisterView from './commonLoginRegisterView';
+import * as CONSTANT from '../constants/constant';
+
+export default class LoginView extends CommonLoginRegisterView {
+  constructor() {
+    super();
+
+    this.parentElement = document.querySelector('.form');
+    this.loginPage = document.getElementById('loginPage');
```

Comment:
Should not depend on HTML element to detect your page

Reply:
Fixed.